validator-server/src/test/kotlin/science/numcompass/conformance/fhir/chain/TerminologyServiceKtTest.kt

```diff
@@ -0,0 +1,76 @@
+package science.numcompass.conformance.fhir.chain
+
+import assertk.assertThat
+import assertk.assertions.isFalse
+import assertk.assertions.isTrue
+import org.junit.jupiter.api.Test
+
+internal class TerminologyServiceKtTest {
+
+    private val snomedSystemUrl = "http://snomed.info/sct"
+
+    @Test
+    fun `An arbitrary code not declared as SNOMED is not skipped`() {
+        val testString = "12344:4356=4545"
+        val valResult = isPostCoordinateSnomed("http://loinc.org", testString)
+        assertThat(valResult).isFalse()
+    }
+
+    @Test
+    fun `A normal, single SNOMED code (integer) declared as SNOMED is not skipped`() {
+        val testString = "410594000"
+        val valResult = isPostCoordinateSnomed(snomedSystemUrl, testString)
+        assertThat(valResult).isFalse()
+    }
+
+    @Test
+    fun `An alphanumeric code declared as SNOMED is not skipped`() {
+        val testString = "ad67886asjd"
+        val valResult = isPostCoordinateSnomed(snomedSystemUrl, testString)
+        assertThat(valResult).isFalse()
```

Comment:
Why is that false here? 🤔
It's not skipped. 
And it's not matching regex .. shouldn't it be true?

Reply:
I don't quite understand that? -  it does match the regex.

Reply:
Makes, sense. My bad